src/tiledb/cloud/soma/ingest.py

```diff
@@ -136,15 +218,18 @@ def build_ingest_workflow_graph(
         )
 
         for entry_input_uri in vfs.ls(input_uri):
+            logging.debug(f"ENUMERATOR ENTRY_INPUT_URI={entry_input_uri}")
```

Comment:
few more logging statements to clean up

Reply:
@thetorpedodog I fail to see what's dirty & needing "cleanup" but I'll put `%r` in there if that's the ask